spec/services/assign_fika_score_spec.rb

```diff
@@ -0,0 +1,52 @@
+RSpec.describe 'AssignScoreService', type: :request do
+  let!(:participant_1) { create(:participant, start_date: Time.now, fika_score: 0) }
+  let(:admin) { create(:admin) }
+  let(:credentials) { admin.create_new_auth_token }
+
+  describe 'when the participants have a delta between their start dates of under 5 years' do
```

Comment:
I never thought I would say this, but maybe this language is a little too mathsy if we want our specs to serve as documentation? I suggest changing `delta` here to `difference`.